games-board/cockatrice/cockatrice-20230914.ebuild

```
@@ -0,0 +1,64 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake
+
+MY_PV="2023-09-14-Release-2.9.0"
+
+DESCRIPTION="Open-source multiplatform software for playing card games over a network"
+HOMEPAGE="https://github.com/Cockatrice/Cockatrice"
+SRC_URI="https://codeload.github.com/Cockatrice/Cockatrice/tar.gz/refs/tags/${MY_PV} -> ${P}.tar.gz"
```

```suggestion
SRC_URI="https://github.com/Cockatrice/Cockatrice/archive/refs/tags/${MY_PV}.tar.gz -> ${P}.tar.gz"
```

Don't see people using codeload.github urls for tarballs usually.

---

Fine I've upgraded the SRC_URI